src/tools/tests/goldens/generated-schemas.wasm.kt

```diff
@@ -113,6 +113,138 @@ typealias Gold_Data_Ref_Spec = GoldInternal1_Spec
 typealias Gold_Alias = GoldInternal1
 typealias Gold_Alias_Spec = GoldInternal1_Spec
 
+class Gold_QCollection() : WasmEntity {
```

Comment:
Same comments apply here

Reply:
This is code gen'd, resolving as duplicate